circle.yml

```diff
@@ -1,8 +1,7 @@
 machine:
     python:
-        version: 2.7.11
-  # post:
-  #   - pyenv global 2.7.11 3.4.3
+        # version: 2.7.11
+        version: 3.5.1
```

Comment:
Trying to get circleci to run multiple versions of python was a huge pain. I tried all following 2 suggestions and neither worked:
    1. circleci docs: https://circleci.com/docs/language-python/
    2. tox plugin: https://github.com/samstav/circleci-python-sandbox
 
Also, circleci uses pyenv to install all [other python versions](https://circleci.com/docs/build-image-trusty/#python) other than the default: `2.7.11`. When running matplotlib in pyenv virtualenv, the GUI frameworks that matplotlib uses for interactive figures have some issues with the virtual environments. http://matplotlib.org/faq/virtualenv_faq.html. 
![screen shot 2016-09-19 at 6 08 47 pm](https://cloud.githubusercontent.com/assets/4450007/18654225/345d3e30-7e94-11e6-8852-a7c19689516f.png)
Switching to travis for now!